source/manual/data-gov-uk-operations.html.md

```diff
@@ -24,18 +24,24 @@ This will populate a new index and rotate the `dataset-staging` alias to point t
 
 ## Sync from [CKAN]
 
-This is done automatically using Sidekiq Scheduler. You can run manually to populate your local DB.
+### Perform a full re-sync
```

Comment:
Should we bump the last reviewed date for this file?

Reply:
Good point.  Updated the date now.